mne/preprocessing/maxwell.py

```diff
@@ -260,8 +296,9 @@ def _sss_basis(origin, coils, int_order, ext_order):
     origin : ndarray, shape (3,)
         Origin of the multipolar moment space in millimeters
     coils : list
-        List of MEG coils. Each should contain coil information dict. All
-        position info must be in the same coordinate frame as 'origin'
+        List of MEG coils. Each should contain coil information dict specifying
+        position, normals, weights, number of integration points and channel
+        type. All position info must be in the 'origin' coordinate frame
```

Comment:
'origin' coord frame? You mean 'head' or 'device'? If it is 'origin' as in the origin of the SSS expansion, can you add a comment saying how these are related to `head` and `device`?